src/components/WeatherDetailHumidity.tsx

```diff
@@ -6,155 +6,155 @@ interface Prop {
   data: any;
 }
 
-function WeatherDetailHumidity(props: Prop) {
+function WeatherDetailHumidity({ day, data }: Prop) {
   return (
     <Center display="block" mr="auto" ml="auto" mt="2rem">
       <HStack textAlign="center">
         <WeatherHourBlockHumidity
-          day={props.day}
+          day={day}
           normalHour="12:00"
           militaryHour={0}
-          data={props.data}
+          data={data}
         />
         <WeatherHourBlockHumidity
-          day={props.day}
+          day={day}
           normalHour="1:00"
           militaryHour={1}
-          data={props.data}
+          data={data}
         />
         <WeatherHourBlockHumidity
-          day={props.day}
+          day={day}
           normalHour="2:00"
           militaryHour={2}
-          data={props.data}
+          data={data}
         />
         <WeatherHourBlockHumidity
-          day={props.day}
+          day={day}
           normalHour="3:00"
           militaryHour={3}
-          data={props.data}
+          data={data}
         />
```

Comment:
Try to reduce the duplication in this code. `WeatherHourBlockHumidity` could be made via a map or an array for instance. Let me know if you want to pair on this.